entc/gen/template/dialect/sql/globals.tmpl

```diff
@@ -18,6 +18,9 @@ func insertLastID(ctx context.Context, tx dialect.Tx, insert *sql.InsertBuilder)
 		}
 		defer rows.Close()
 		if !rows.Next() {
+			if rows.Err() != nil {
+				return 0, rows.Err()
+			}
 			return 0, fmt.Errorf("no rows found for query: %v", query)
```

Comment:
You can remove this change and re-run codegen, I just deleted this template code.